sys/kern/vfs_mount.c

```diff
@@ -419,8 +419,8 @@ kern_nmount(struct thread *td, struct iovec * __capability iovp, u_int iovcnt,
 	flags = flags32;
 
 	AUDIT_ARG_FFLAGS(flags);
-	CTR4(KTR_VFS, "%s: iovp %#lx with iovcnt %d and flags %d", __func__,
-	    iovp, (u_long)iovcnt, flags);
+	CTR4(KTR_VFS, "%s: iovp %lp with iovcnt %d and flags %d", __func__,
+	    (uintcap_t)iovp, iovcnt, flags);
```

Comment:
hm I'd expect this to fail for non-CHERI since %p doesn't allow unsigned long. Does `%lp` with a `void * __capability` arg work?